### ISSUE TITLE
Operant?

### ISSUE THREAD

**Issue by spl** [leanprover/tutorial#175]
The word “operant” is used in a number of places in the tutorial text, and it is not defined.

I've never seen the word before (as a native English speaker who has read quite a bit of programming language research material), and Google doesn't really provide a useful meaning that seems applicable in this situation. I infer the meaning to be either “in scope” (as it applies to notation) or “in use” (as it applies to attributes).

I wonder if “operant” is the best word to use, because it seems to be obscure and not well defined. I would suggest using different terms where appropriate.

**Comment by avigad** [comment]
I didn't realize that I was using it so much! The dictionary meaning is here:

  http://www.merriam-webster.com/dictionary/operant

It can be used to mean "in effect," "in use," "in scope," "in play," etc. I'll change them to "in effect" or "in use."

**Comment by spl** [comment]
Yes, I did see the dictionary meaning, but I think the dictionary is leaving out some context for its use. The main context I found was in psychology and social science. Whereas, it felt like the tutorial was using it in a technical sense that, perhaps, was not implied by the dictionary. Given that impression, I think a less obscure vocabulary choice is warranted. But, out of curiosity, have you seen the word used elsewhere?
